class-two-factor-core.php

```diff
@@ -90,6 +90,9 @@ public static function add_hooks( $compat ) {
 		// Run only after the core wp_authenticate_username_password() check.
 		add_filter( 'authenticate', array( __CLASS__, 'filter_authenticate' ), 50 );
 
+		// Run as late as possible to disable sending authentication cookies.
```

Comment:
```suggestion
		// Run as late as possible to prevent other plugins from unintentionally bypassing.
```

The "what" is obvious from the callback name and its docblock. I think the "why" would be more useful here.